index.html

```diff
@@ -42,47 +43,58 @@
 }
 </pre>
 
-## Dependencies
-The <dfn>`ReadableStream`</dfn> and <dfn>`WritableStream`</dfn> interfaces are
-defined in [[!STREAMS]].
+## Terminology and conventions
 
-The <dfn>`Promise`</dfn> and <dfn>`DOMException`</dfn> interfaces are defined
-in [[!DOM]].
+The terms
+<dfn data-cite="!STREAMS#rs-model">`ReadableStream`</dfn> and
+<dfn data-cite="!STREAMS#ws-model">`WritableStream`</dfn>
```

Comment:
```suggestion
<code><dfn data-cite="STREAMS#ws-model">WritableStream</dfn></code>
```

Same pattern for other dfns that link to interfaces. 
ReSpec will automatically add code around any `<a>`s.

Reply:
done